geiger/src/graph/Graph.js

```diff
@@ -51,6 +54,10 @@ export default class Graph extends Events {
     this._in = {};
   }
 
+  /**
+   * Add cell to the graph based on its type.
+   * @param {!Cell} cell
```

Comment:
What's `Cell`? This seems to be something other than `Node`.

Reply:
`Cell` is a base class for `Node` and `Edge`.